ci/before_deploy.sh

```diff
@@ -8,6 +8,10 @@ set -ex
 
 # Generate artifacts for release
 mk_artifacts() {
+    if ! grep -q '^lto.*=.*true' Cargo.toml 2>/dev/null; then
+       sed  -i '/\[profile.release\]/a lto = true' Cargo.toml
```

Comment:
My guess is that this doesn't work on macOS.

Reply:
It does not, both because of the missing `-i` extension and because the command can't be written that way. Something like this should work with both GNU and BSD `sed`:

```
sed -i.bak $'/\\[profile\.release\\]/a\\\nlto = true\n' Cargo.toml
```

The `grep` pattern is also not very precise